src/features/list-view/ListView.js

```diff
@@ -28,21 +33,34 @@ type CellRendererArgs = {|
 
 class ListView extends React.PureComponent<Props> {
     cellRenderer = ({ columnIndex, key, rowIndex, style }: CellRendererArgs) => {
-        const { getGridCell, getGridHeader } = this.props;
+        const { getGridCell, getGridHeader, getGridHeaderSort, onSortChange } = this.props;
 
         if (rowIndex === 0) {
             const displayName = getGridHeader(columnIndex);
+            const sortDirection = getGridHeaderSort && getGridHeaderSort(columnIndex);
+            const isSortAsc = sortDirection === SORT_ORDER_ASCENDING;
+            const className = classNames({
+                'bdl-ListView-is-sort-asc': isSortAsc,
+            });
+
             return (
-                <div className="list-view-column-header" key={key} style={style}>
+                <button
+                    className="bdl-ListView-list-view-column-header"
```

Comment:
So typically the SUIT syntax goes
$bdl-Button 
followed by lowercase descriptor
-content
followed by camelCase for the final descriptor
-backgroundColor
$bdl-Button-content-backgroundColor.

In this case I think you className should be
bdl-ListView-columnHeader